lara-typescript/src/section-authoring/components/managed-interactive-preview.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import classNames from "classnames";
+import { ISectionItem } from "../api/api-types";
+import { usePageAPI } from "../hooks/use-api-provider";
+import { InteractiveAuthoringPreview } from "../../page-item-authoring/common/components/interactive-authoring-preview";
+
+import "./managed-interactive-preview.scss";
+
+export interface IManagedInteractivePreviewProps {
+  pageItem: ISectionItem;
+}
+
+export const ManagedInteractivePreview: React.FC<IManagedInteractivePreviewProps> = ({
+  pageItem
+  }: IManagedInteractivePreviewProps) => {
+  const {
+    aspectRatio, aspectRatioMethod, authoredState, customAspectRatioMethod,
+    id, interactiveItemId, isFullWidth, name, libraryInteractiveId,
+    linkedInteractives, urlFragment
+   } = pageItem.data as any;
```

Comment:
It would b good to add a little typing here if we could.

This `data` thing is probably typed somewhere already, if there is some problem with type field overlap, we could use a `Partial<IDataType>` in which all the fields are considered optional, that would still be an improvement.